proto/data.proto

```diff
@@ -252,6 +252,22 @@ message Transaction {
   optional NodeList certain_nodes = 12 [(gogoproto.nullable) = false];
 }
 
+// Lease contains information about leader leases including the
+// expiration and lease holder.
+message Lease {
+  // The expiration is in unix nanos and is set according to the wall
+  // clock at the lease requestor / grantee.
+  optional int64 expiration = 1 [(gogoproto.nullable) = false];
```

Comment:
The lease should include both an expiration and a duration. The requesting leader uses the expiration and the followers use the duration.